README.md

```diff
@@ -13,15 +12,28 @@ Image basics (0:30 hours)
 Working with skimage (0:30 hours)
 Drawing and Bitwise Operations (2:30 hours)
 Creating Histograms (1:00 hours)
-Blurring Images (1:30 hours)
+Summary lecture(1:00 hours)
+
+Day 2:
+Blurring Images (1:00 hours)
 Thresholding (1:00 hours)
 Connected Components Analysis (1:00 hours)
-Bonus: Affine and Transformations (1:00 hours)
 Capstone Challenge (1:00 hours)
+Bonus: Affine and Transformations (1:00 hours)
+Summary, wrap up (1:00 hours)
+
 
-Wrap up
 Exercises document
 
 The course will also be taught [interally](internal_workshop.md).
 
+## Environment:
+    To work with the correct packages:
+    Install all Python packages required, using conda and the
+    `environment.yml` file.
+
+    * The command for Windows/Anaconda users will be:
+     `conda env create -f environment.yml`.
```

Comment:
```suggestion
To work with the correct packages:
Install all Python packages required, using conda and the`environment.yml` file.

The command for Windows/Anaconda users will be:
```
conda env create -f environment.yml
```
```

Reply:
I don't think this suggestion works :/ sorry.
Here, I would take all text out of the code block, and only include the `conda env create...` command in a code block.